apps/src/StudioApp.js

```diff
@@ -387,6 +397,7 @@ StudioApp.prototype.isUsingBlockly = function () {
 StudioApp.prototype.handleSharing_ = function (options) {
   // 1. Move the buttons, 2. Hide the slider in the share page for mobile.
   var belowVisualization = document.getElementById('belowVisualization');
+  var visualization = document.getElementById('visualization');
```

Comment:
Caught by Strict Mode: We were using visualization in this function without defining it, probably a missed detail in a method extraction.